### PR TITLE
Added support for 0.9.7 custom RBAC LDAP plugin

### PR DESCRIPTION
**Pull request by richarpw**

### CODE REVIEW

File: scripts/make-guac.sh [hunk 1]
```diff
@@ -376,6 +384,24 @@ then
...
+        (
+            echo "ldap-group-base-dn       ${LDAP_GROUP_BASE},${LDAP_DOMAIN_DN}"
+        ) >> /etc/guacamole/guacamole.properties
+        
+        if [ "$GUAC_VERSION" == "0.9.7" ]
+        then
+            log "Enabling custom RBAC jar for 0.9.7"
+            rm -rf "/etc/guacamole/extensions/*"
+            cd "/etc/guacamole/extensions/"
+            curl -s -O https://s3.amazonaws.com/dicelab-guacamole/guacamole-auth-ldap-0.9.7.jar
```
Comment (lorengordon) [inline scripts/make-guac.sh]: I'd like to add the `|| die` syntax to the curl line.

File: scripts/make-guac.sh [hunk 2]
```diff
@@ -376,6 +384,24 @@ then
...
+        else
+            log "Warning: Unknown RBAC support in this GUAC version, not 0.9.7!"
```
Comment (lorengordon) [inline scripts/make-guac.sh]: bash will throw up on exclamation points in double quotes. Need to use single quotes:

`log "Warning: Unknown RBAC support in this GUAC version, not 0.9.7"'!'`

But I have not yet tried passing an exclamation point through the `log` function, so probably want to test that.

**Comment by lorengordon** [comment]
@richarpw, just two comments, otherwise looks good. Thanks!

File: scripts/make-guac.sh (hunk 1 above)
Comment (richarpw) [inline scripts/make-guac.sh]: Yep, will do that.

File: scripts/make-guac.sh (hunk 2 above)
Comment (richarpw) [inline scripts/make-guac.sh]: Good catch, sorry forgot to test that scenario. Though it did work via log.

Output, "Warning: Unknown RBAC support in this GUAC version, not 0.9.7!"

**Comment by lorengordon** [comment]
:+1: ready to merge?

**Comment by richarpw** [comment]
:+1: Ready freddy

**Pull request merged**